### ISSUE TITLE
parseXmlResponse Error and Error handling

### ISSUE THREAD

**Issue by SebastianMetzger** [abbyysdk/ocrsdk.com#43]
It is pretty cool, that abbyy provides a NodeJS library for their API and so far it also worked well for us!

We currently have issues, that the processImage task is not working for certain users and the error handling of the ocrsdk.js is a bit buggy itself, as the error message is show as [Object object].
So it is hard to debug right now, without monkey patching the library.

Probably there is a missing "toString()" after message or something?
taskDataCallback(new Error(response.error.message), null);

Here is the example from our log file:
![abbyy error](https://cloud.githubusercontent.com/assets/2345498/14603148/62281c8c-056c-11e6-9422-68bca1faca11.PNG)

**Comment by Dmitry-Me** [comment]
No, not `toString()`. This https://github.com/abbyysdk/ocrsdk.com/commit/fa44845d2ee99fdb8ca9b9fe0b1dde116b4c96fe

**Comment by Dmitry-Me** [comment]
@SebastianMetzger Could you please try the changes from https://github.com/abbyysdk/ocrsdk.com/commit/fa44845d2ee99fdb8ca9b9fe0b1dde116b4c96fe ?

**Comment by SebastianMetzger** [comment]
thx, for providing the fix.

I build in my own error logging and did not try yours yet.
Just upgraded it and will report, if it worked!
